### PR TITLE
Base layers and theme in one request

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
@elemoine can you review this pull request ?

File: c2cgeoportal/tests/functional/test_themes.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def _only_name(self, item, attribute='name'):
 
     def _get_filtred_errors(self, themes):
```
Comment (asaunier) [inline c2cgeoportal/tests/functional/test_themes.py]: filtred => filtered

File: doc/developer/webservices.rst [hunk 2]
```diff
@@ -12,6 +12,9 @@ Parameters
 ----------
 
 * ``version``: ``1`` or ``2 ``, API version, default is ``1``.
+* ``sets``: with kind of data we want to get, can be ``all``, ``themes``, ``group``
```
Comment (asaunier) [inline doc/developer/webservices.rst]: "with kind of data" => "which kind of data" (or simply "kind of data" or "type of data")

File: doc/developer/webservices.rst [hunk 3]
```diff
@@ -12,6 +12,9 @@ Parameters
...
+  or ``background_layers``, default is ``all``.
+* ``background_layers_group``: get only one group, used to get the base layers.
```
Comment (asaunier) [inline doc/developer/webservices.rst]: "get only one group, used to get the base layers." => "get the baselayers group"?

Comment (asaunier) [inline doc/developer/webservices.rst]: You should make it more explicit that this param should contain the name of the baselayers group and its default value.

File: doc/developer/webservices.rst [hunk 4]
```diff
@@ -12,6 +12,9 @@ Parameters
...
 * ``group``: get only one group, used to get the base layers.
```
Comment (asaunier) [inline doc/developer/webservices.rst]: ??
For baselayers, shouldn't the user use the ``background_layers_group`` param?

File: doc/developer/webservices.rst [hunk 5]
```diff
@@ -21,22 +24,14 @@ Parameters
...
-.. code:: json
-
-    {
-        items: [<themes>],
-        errors: [<errors>]
-    }
-
-
-Main for group:
+Main for all possible results:
```
Comment (asaunier) [inline doc/developer/webservices.rst]: Main what?

File: travis/status.sh [hunk 6]
```diff
@@ -5,6 +5,9 @@ then
     cd $1
 fi
 
+# checkout po file to don't failed on them
```
Comment (asaunier) [inline travis/status.sh]: Not sure what this sentence means :P

File: doc/developer/webservices.rst (hunk 3 above)
Comment (sbrunner) [inline doc/developer/webservices.rst]: => get the background layers referred by the group.

Comment (sbrunner) [inline doc/developer/webservices.rst]: second reword:

parent group of background layers to get.

File: doc/developer/webservices.rst (hunk 4 above)
Comment (sbrunner) [inline doc/developer/webservices.rst]: Yes, =>  the group to get.

File: doc/developer/webservices.rst (hunk 5 above)
Comment (sbrunner) [inline doc/developer/webservices.rst]: => Base ...

File: travis/status.sh (hunk 6 above)
Comment (sbrunner) [inline travis/status.sh]: I added a script comment to be more understandable

**Pull request merged**